geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/dunit/PartitionedRegionCqQueryDUnitTest.java

```diff
@@ -120,6 +125,109 @@
 
   private static int bridgeServerPort;
 
+  @Test
+  public void testPutAllWithCQLocalDestroy() {
+    VM server1 = getVM(0);
+    VM server2 = getVM(1);
+    VM client = getVM(2);
+
+    final String cqName = "testPutAllWithCQLocalDestroy_0";
+    createServer(server1);
+    createServer(server2);
+    final String host = VM.getHostName();
+    final int port = server2.invoke(() -> getCacheServerPort());
+    createClient(client, port, host);
+    createCQ(client, cqName, cqs[0]);
+
+    int numObjects = 1000;
```

Comment:
Does this need to be that big...If its not needed can we keep it small, say 5 or 10. Makes test to run faster.

Reply:
Yes. It does need a larger number. Small number sometimes does not consistently reproduce the bug.

Reply:
Can you please help me understand why consistency an issue here; in that case how sure we are that 1000 entries will recreate the issue all the time.